### PR TITLE
Updates for ITKv5.2 code cleanups ... ROUND1

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: include/itkTileConfiguration.h [hunk 1]
```diff
@@ -298,8 +298,8 @@ struct ITK_TEMPLATE_EXPORT TileConfiguration
     }
     std::getline(ss, temp, '(');
 
-    using PointType = itk::Point<double, Dimension>;
-    PointType p;
+    using PointTypelocal = itk::Point<double, Dimension>;
```
Comment (dzenanz) [inline include/itkTileConfiguration.h]: Commit message misleading.

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -397,7 +397,7 @@ itk_add_test(NAME itkMontagePCMFilesS200_32_33
     74.09566
     7.614863315
   )
-endif(ITKMontage_EnableFailingTests)
```
[the remark this answers is not in the document]
Comment (hjmjohnson) [reply test/CMakeLists.txt]: @dzenanz This is against modern CMake style.  I would prefer 

```
endif() # ITKMontage_EnableFailingTests
```

[the remark this answers is not in the document]
Comment (dzenanz) [reply test/CMakeLists.txt]: That is fine too.

File: test/CMakeLists.txt [hunk 3]
```diff
@@ -537,7 +537,7 @@ if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/Input/Ti7/Region2_150_Mosaic36Flat/36/1323.
       0.0
       0.0
     )
-  endif(ITKMontage_EnableFailingTests)
```
[the remark this answers is not in the document]
Comment (hjmjohnson) [reply test/CMakeLists.txt]: FYI: The very long block is only 12 lines long.

[the remark this answers is not in the document]
Comment (dzenanz) [reply test/CMakeLists.txt]: The block might have been longer at a previous point in time 😄

**Comment by dzenanz** [comment]
KWStyleTest fails, as well as linter.

**Comment by dzenanz** [comment]
I don't get [`d:\a\1\s\include\itkTileMergeImageFilter.hxx(359): error C2440: 'initializing': cannot convert from 'unsigned __int64' to 'unsigned long &'`](https://open.cdash.org/viewBuildError.php?buildid=6371322) when I build locally.

**Comment by dzenanz** [comment]
After applying #159, I also get cannot convert error locally. I will see if I can fix it.

**Pull request merged**